bus-mapping/src/evm/opcodes.rs

```diff
@@ -464,6 +477,56 @@ pub fn gen_end_tx_ops(state: &mut CircuitInputStateRef) -> Result<ExecStep, Erro
         },
     );
 
+    // handle tx receipt tag
+    state.push_op(
+        &mut exec_step,
+        RW::READ,
+        TxReceiptOp {
+            tx_id: state.tx_ctx.id(),
+            field: TxReceiptField::PostStateOrStatus,
+            value: call.is_persistent as u64,
+        },
+    );
+
+    let log_id = exec_step.log_id;
+    state.push_op(
+        &mut exec_step,
+        RW::READ,
+        TxReceiptOp {
+            tx_id: state.tx_ctx.id(),
+            field: TxReceiptField::LogLength,
+            value: log_id as u64,
```

Comment:
Is `log_id` zero indexed? I mean, for the first log, is `log_id == 0`?

In that case, we should have `LogLength == log_id + 1` right?